doc/openapi.yaml

```diff
@@ -0,0 +1,78 @@
+openapi: 3.0.0
+info:
+  title: SDM SQL Schema Generation
+  version: 0.1.0
+  description: FastAPI application for SDM Quality Testing
+
+paths:
+  /version:
+    get:
+      summary: Get version information
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  doc:
+                    type: string
+                  git_hash:
+                    type: string
+                  version:
+                    type: string
+                  release_date:
+                    type: string
+                  uptime:
+                    type: string
+
+  /generate:
+    post:
+      summary: Generating a SQL Schema
```

Comment:
summary: Generating a SQL Schema
  description: Generate the corresponding SQL Schema of a provided Data Model in the Smart Data Model repository
  operationId: generateSQLSchema